common/src/main/java/com/wynntils/features/MythicFoundFeature.java

```diff
@@ -36,26 +38,54 @@ public void onMythicFound(MythicFoundEvent event) {
+
         ItemStack itemStack = event.getMythicBoxItem();
-        Optional<GearBoxItem> wynnItem = Models.Item.asWynnItem(itemStack, GearBoxItem.class);
-        if (wynnItem.isEmpty()) return;
-
-        if (showDryStreakMessage.get() && wynnItem.get().getGearType() != GearType.MASTERY_TOME) {
-            McUtils.sendMessageToClient(Component.literal("Dry streak broken! Found a ")
-                    .withStyle(ChatFormatting.LIGHT_PURPLE)
-                    .append(event.getMythicBoxItem().getHoverName())
-                    .append(Component.literal(" in chest ")
-                            .withStyle(ChatFormatting.LIGHT_PURPLE)
-                            .append(Component.literal("#" + Models.LootChest.getOpenedChestCount())
-                                    .withStyle(ChatFormatting.GOLD)))
-                    .append(Component.literal(" after ")
-                            .withStyle(ChatFormatting.LIGHT_PURPLE)
-                            .append(Component.literal(String.valueOf(Models.LootChest.getDryCount()))
-                                    .withStyle(ChatFormatting.DARK_PURPLE)))
-                    .append(Component.literal(" dry chests and "))
-                    .append(Component.literal(String.valueOf(Models.LootChest.getDryBoxes()))
-                            .withStyle(ChatFormatting.DARK_PURPLE))
-                    .append(Component.literal(" dry boxes.")));
+
+        // Normal loot chest reward
+        Optional<GearBoxItem> gearBoxItem = Models.Item.asWynnItem(itemStack, GearBoxItem.class);
+        if (gearBoxItem.isPresent()) {
+            if (gearBoxItem.get().getGearType() != GearType.MASTERY_TOME) {
+                sendNormalDryStreakMessage(
+                        StyledText.fromComponent(event.getMythicBoxItem().getHoverName()));
+            }
+            return;
+        }
+
+        // Lootrun reward
+        Optional<GearItem> gearItem = Models.Item.asWynnItem(itemStack, GearItem.class);
+        if (gearItem.isPresent()) {
+            sendLootrunDryStreakMessage(
+                    StyledText.fromComponent(event.getMythicBoxItem().getHoverName()));
         }
     }
+
+    private void sendLootrunDryStreakMessage(StyledText itemName) {
+        McUtils.sendMessageToClient(Component.literal("Dry streak broken! Found an ")
+                .withStyle(ChatFormatting.LIGHT_PURPLE)
+                .append(itemName.getComponent())
+                .append(Component.literal(" after ")
+                        .withStyle(ChatFormatting.LIGHT_PURPLE)
+                        .append(Component.literal(Models.LootChest.dryPulls.get() + " pulls")
+                                .withStyle(ChatFormatting.GOLD)))
+                .append(Component.literal(" without a mythic.").withStyle(ChatFormatting.LIGHT_PURPLE)));
+    }
+
+    private static void sendNormalDryStreakMessage(StyledText itemName) {
+        McUtils.sendMessageToClient(Component.literal("Dry streak broken! Found a ")
```

Comment:
```suggestion
        McUtils.sendMessageToClient(Component.literal("Dry streak broken! Found an ")
```

To make it the same as the other message